### ISSUE TITLE
[Feature]: Support Microsoft Teams for Notifications

### ISSUE THREAD

**Issue by fersy02** [apisec-inc/Feature-Request#79]
### Please describe your problem?

Currently, we only support Slack notifications, and most of our customers are Microsoft shop and have reached out to us to support Microsoft Teams(webhook) for easy integration.

### How important is this feature?

Nice to Have

### Describe the solution you'd like?

Extending support for notifications to Microsoft Teams using Webhook.

**Comment by atef-aa** [comment]
@fersy02 - Is this blocking a deal? If not, we can pick it up in once some of the sales and CS priority work is complete.
